source/Calamari.Terraform.Tests/CommandsFixture.cs

```diff
@@ -26,13 +26,12 @@
 
 namespace Calamari.Terraform.Tests
 {
-    [TestFixture("0.11.15")]
-    [TestFixture("0.13.0")]
-    [TestFixture("1.0.0")]
+    [TestFixture("0.13.7")]
+    [TestFixture("1.8.5")]
```

Comment:
I don't think it's a big issue if the tested Terraform versions change, but happy to hear dissenting opinions